### DISCUSSION TITLE
Terminal emulator window is opened with a significant delay

### DISCUSSION THREAD

**Discussion by sodamouse** [ghostty-org/ghostty#4164] [Q&A] [answered]
Debian GNU/Linux 12 (bookworm) x86_64
dwm window manager (X11, xinit)
Terminal emulator window is created after several minutes of waiting when launching ghostty compiled from source (tag v1.0.0).

The Following output is generated, then application enters main loop, and I have to wait several minutes before the terminal window is opened.

> info: ghostty version=1.0.0
> info: ghostty build optimize=ReleaseFast
> info: runtime=apprt.Runtime.gtk
> info: font_backend=font.main.Backend.fontconfig_freetype
> info: dependency harfbuzz=8.4.0
> info: dependency fontconfig=21402
> info: renderer=renderer.OpenGL
> info: libxev backend=main.Backend.io_uring
> info(os): setlocale from env result=en_CA.UTF-8
> info(gtk): GTK version build=4.8.3 runtime=4.8.3
> info: optional config file not found, not loading path=/home/soda/.config/ghostty/config
> info(config): default shell source=env value=/bin/bash
> info(sentry): sentry envelope does not contain crash, discarding
> info(gtk): libadwaita version build=1.2.2 runtime=1.2.2
>

**Comment by rhodes-b** [comment]
What is significant?

Also you have old versions of GTK and adwaita which could be part of this 

GTK / Ghostty will attempt to talk to DBUS, if It cannot this will cause a delay during booting. That seems like your case since you are running a very minimal wm and not a full DE (at least from your description)

Anyways this is pretty known and is a dupe of multiple? other discussions

**Reply by rhodes-b** [reply, marked as answer]
https://github.com/ghostty-org/ghostty/discussions/3720
https://github.com/ghostty-org/ghostty/discussions/2978
